src/Akka.Persistence.SqlServer.Tests/Batching/BatchingSqlServerEventsByPersistenceIdSpec.cs

```diff
@@ -1,14 +1,12 @@
-﻿//-----------------------------------------------------------------------
+﻿// -----------------------------------------------------------------------
 // <copyright file="BatchingSqlServerEventsByPersistenceIdSpec.cs" company="Akka.NET Project">
-//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
-//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+//      Copyright (C) 2013 - 2019 .NET Foundation <https://github.com/akkadotnet/akka.net>
```

Comment:
This code is all original work from the Akka.NET team, so no need to credit Lightbend. Also - changed copyright header to target .NET Foundation, which is accurate.